### DISCUSSION TITLE
Problemas con el repositorio local y del servidor

### DISCUSSION THREAD

**Discussion by Axelito02** [sebasvelasco353/FDP-2022_01#80] [General]
Profe intento clonar el repositorio al local de nuevo pero esta clonando normal y luego aparece esto....
![image](https://user-images.githubusercontent.com/98567345/156475403-800dbfed-91c0-4db2-a109-d76bdc1d8292.png)
... y la carpeta del repositorio no se clona completo queda vacío con solo una carpeta llamada ".git" 
![image](https://user-images.githubusercontent.com/98567345/156475507-f684252f-1e70-402e-a3e1-e0e6c7d6ecb4.png)
Profe como hago para solucionar eso.
![image](https://user-images.githubusercontent.com/98567345/156475730-d0a69996-ce07-4700-86eb-aec2a8ecd54f.png)

**Comment by sebasvelasco353** [comment]
Ando leyendo posibles soluciones a eso, mientras tanto tengo una hipotesis sobre que puede estar causando eso... en internet hay gente que dice que puede estar pasando por temas de carpetas y nombres.
Voy a intentar hacer un cambio en los nombres de las carpetas para que vuelvas a intentar hacer el proceso de pull y clone y me cuentes si el problema cabia o se desaparece... te escribo por aca cuando lo haga para que lo intentes y me cuentes que pasa dale?

**Reply by sebasvelasco353** [reply]
el problema en resumen: hay palabras y grupos de caracteres reservados en los file systems, puede ser que al tener un punto en una carpeta se esten usando combinaciones prohibidas en los nombres... voy a renombrar las carpetas y ordenar el repo para solucionarlo.

**Reply by sebasvelasco353** [reply]
el link de donde encontre la posible razon y una solucion es este por si quieres leerlo y entender un poco mas. 

https://github.com/MicronOxford/cockpit/issues/704
